File: b/Teleperformance.DataIngestion.API/Teleperformance.DataIngestion.Models/DTOs/v1.0/FileConfiguration/FlpConvertParquetRequestDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Teleperformance.DataIngestion.Models.Entities.v1._0;

namespace Teleperformance.DataIngestion.Models.DTOs.v1._0.FileConfiguration
{
    public class FlpConvertParquetRequestDto
    {
        public string FlpConfigurationId { get; set; }
        public string ProcessName { get; set; }
        public BlobClientDetails BlobClients { get; set; }
        public OnPremFileLocation OnPremFileLocation { get; set; }
    }
}

File: f/src/app/core/models/FlpConvertParquetRequestDto.ts
export interface FlpConvertParquetRequestDto {
    flpConfigurationId : string;
    processName : string;
    blobClients : BlobClients
    //onPremFileLocation : OnPremFileLocation
}

export interface BlobClients {
    uploadedId : string;
    uri : string;
    accountName : string;
    blobContainerName : string;
    name : string;
    canGenerateSasUri : boolean;
    
}

export interface OnPremFileLocation {
    UploadedId : string;
    FileUrl : string;
}